hockeysdk/src/main/java/net/hockeyapp/android/metrics/Sender.java

```diff
@@ -125,6 +125,9 @@ protected void sendAvailableFiles() {
             if ((persistedData != null) && (connection != null)) {
                 send(connection, fileToSend, persistedData);
             }
+            if (connection != null) {
```

Comment:
Isn't this problematic?!
`send()` opens the connection and then, it gets `disconnected()` right away? Shouldn't we do this once we received the response?